CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at julianagustin78@gmail.com. All
+reported by contacting the project team at julian.pasquale@rootstrap.com. All
```

Comment:
Maybe we could spin up a special address for this, like `coc@rootstrap.com` something like that?

Reply:
Sure! I really don't know which address we should use. Other projects like [arqo](https://github.com/rootstrap/arqo/blob/master/CODE_OF_CONDUCT.md) and [apple_auth](https://github.com/rootstrap/apple_auth/blob/master/CODE_OF_CONDUCT.md) don't have any Rootstrap address either

Reply:
Let's leave it like this, work towards having the new inbox address and then make the needed updates. 😎